python/example/ring.py

```diff
@@ -59,10 +59,19 @@ def event_generators(self, gid):
 decomp = arbor.partition_load_balance(recipe, context)
 print(f'{decomp}')
 
+hint = arbor.partition_hint()
+hint.prefer_gpu = False
+hint.gpu_group_size = 1
```

Comment:
It is a bit odd to set gpu_group_size after setting `prefer_gpu` to false.
It also goes against the recommended approach of using many cells per group on the GPU (and an example should illustrate best practices.)

Maybe set it to `True`, then set `gpu_group_size=1000`?

Reply:
True. Done.